src/TheThingsNetwork.cpp

```diff
@@ -93,29 +84,30 @@ bool TheThingsNetwork::sendCommand(String cmd, const byte *buf, int length, int
   return waitForOK(waitTime);
 }
 
-void TheThingsNetwork::reset(bool adr, int sf, int fsb) {
+void TheThingsNetwork::reset(bool adr) {
   #if !TTN_ADR_SUPPORTED
     adr = false;
   #endif
```

Comment:
We can get rid of `TTN_ADR_SUPPORTED` and get rid of this check, because we support ADR now